src/lib/components/AppInfo.svelte

```diff
@@ -84,7 +84,15 @@
           <div class="detail-row">
             <span class="label">developer</span>
             <span class="separator">::</span>
-            <span class="value">{APP_INFO.github}</span>
+            <a
+                href={APP_INFO.github}
```

Comment:
let's also change the url to neohtop's repo instead

Reply:
Done!

Reply:
for next times you can resolve the conversation and re-request review again (would get notified on mobile).. but good job @Pinguin2001